app/services/GuidanceService.scala

```diff
@@ -107,7 +109,7 @@ class GuidanceService @Inject() (
               .fold(err => {
                 logger.warn(s"Failed to parse process with error $err")
                 Left(InvalidProcessError)
-              }, pages => Right(pages.head.url))
+              }, pages => Right((pages.head.url, process.meta.processCode.getOrElse(process.meta.id))))
```

Comment:
Cpuld add a method on Process to conditionally return the processCode or id depending on what is available (as titlePhrase), stops the spread of the uncertainty ;)

Reply:
Fixed